### PR TITLE
Convert Browser sandbox to Cloud sandbox.

### PR DESCRIPTION
**Pull request by Ekwuno**
This tutorial shows how to convert browser sandbox to a  Cloud sandbox.

### CODE REVIEW

File: packages/projects-docs/pages/tutorial/convert-browser-sandbox-cloud.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+There are a couple of prompts for moving a sandbox in the browser sandbox UI. For example, if you want to access the terminal we suggest you a Cloud sandbox as the experience is much better there.
+
+![Connect from terminal](/convert-from-terminal.png)
+
+### Access Docker support from Browser sandbox
+
+You can access Docker support from a browser sandbox by clicking on the Docker icon in the menu on the left. By doing so your Sandbox will be converted to a Cloud sandbox. 
+
+![docker support from browser sandbox](/docker-support-browser-sandbox.png)
+
```
Comment (filipelima18) [inline packages/projects-docs/pages/tutorial/convert-browser-sandbox-cloud.mdx]: I'm taking some time to review this.

Comment (filipelima18) [reply packages/projects-docs/pages/tutorial/convert-browser-sandbox-cloud.mdx]: @Ekwuno - WDYT about these changes?

**Review by filipelima18 — COMMENTED** [review]
Made some changes. WDYT @Ekwuno ?

File: packages/projects-docs/pages/tutorial/convert-browser-sandbox-cloud.mdx (hunk 1 above)
Comment (Ekwuno) [reply packages/projects-docs/pages/tutorial/convert-browser-sandbox-cloud.mdx]: @filipelima18  looks good!

**Pull request merged**